### PR TITLE
Single contact is now clickable in 'My Contacts'

### PR DESCRIPTION
**Pull request by Justus-Becker-Rabbenstein**
- Feature/clickablemycontacts
- new page: pages/singleProfile
- displays a single clicked Profile from 'My Contacts'
- hover effect on pages in my contacts to make visisble, that they're clickable
- some design refactoring that was broken

### CODE REVIEW

File: pages/singleProfile/index.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  );
+};
+
+export default singleProfile;
+
+const ContainerDiv = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 10px;
```
Comment (stefankorder) [inline pages/singleProfile/index.js]: Please use relative units at this point.

File: pages/singleProfile/index.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+  margin-left: 10%;
+  width: 80%;
+  gap: 3px;
```
Comment (stefankorder) [inline pages/singleProfile/index.js]: Please use relative units at this point.

File: pages/singleProfile/index.js [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+  backdrop-filter: blur(2.2px);
+  -webkit-backdrop-filter: blur(2.2px);
+  border: 1px solid rgba(255, 255, 255, 0.5);
+`;
+const ContainerTextareaName = styled.textarea`
+  background-image: url("icon_name.svg");
+  background-repeat: no-repeat;
+  background-position: left;
+  background-size: contain;
+  padding-left: 40px;
```
Comment (stefankorder) [inline pages/singleProfile/index.js]: Please use relative units at this point.

File: pages/singleProfile/index.js [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+  border-radius: 20px;
+`;
+const ContainerTextareaAddress = styled.textarea`
+  background-image: url("icon_address.svg");
+  background-repeat: no-repeat;
+  background-position: left;
+  background-size: contain;
+  padding-left: 40px;
```
Comment (stefankorder) [inline pages/singleProfile/index.js]: Please use relative units at this point.

File: pages/singleProfile/index.js [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+  border-radius: 20px;
+`;
+const ContainerTextareaEmail = styled.textarea`
+  background-image: url("icon_email.svg");
+  background-repeat: no-repeat;
+  background-position: left;
+  background-size: contain;
+  padding-left: 40px;
```
Comment (stefankorder) [inline pages/singleProfile/index.js]: Please use relative units at this point.

File: pages/singleProfile/index.js [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+  border-radius: 20px;
+`;
+const ContainerTextareaPhone = styled.textarea`
+  background-image: url("icon_phone.svg");
+  background-repeat: no-repeat;
+  background-position: left;
+  background-size: contain;
+  padding-left: 40px;
```
Comment (stefankorder) [inline pages/singleProfile/index.js]: Please use relative units at this point.

File: pages/singleProfile/index.js [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+  border-radius: 20px;
+`;
+const ContainerTextareaWebsite = styled.textarea`
+  background-image: url("icon_website.svg");
+  background-repeat: no-repeat;
+  background-position: left;
+  background-size: contain;
+  padding-left: 40px;
```
Comment (stefankorder) [inline pages/singleProfile/index.js]: Please use relative units at this point.

**Review by stefankorder — CHANGES_REQUESTED** [review]
LGTM! Please take a look at my comments.

File: pages/singleProfile/index.js (hunk 1 above)
Comment (Justus-Becker-Rabbenstein) [reply pages/singleProfile/index.js]: Fixed.

File: pages/singleProfile/index.js (hunk 3 above)
Comment (Justus-Becker-Rabbenstein) [reply pages/singleProfile/index.js]: Fixed.

File: pages/singleProfile/index.js (hunk 4 above)
Comment (Justus-Becker-Rabbenstein) [reply pages/singleProfile/index.js]: Fixed.

File: pages/singleProfile/index.js (hunk 5 above)
Comment (Justus-Becker-Rabbenstein) [reply pages/singleProfile/index.js]: Fixed.

File: pages/singleProfile/index.js (hunk 6 above)
Comment (Justus-Becker-Rabbenstein) [reply pages/singleProfile/index.js]: Fixed.

File: pages/singleProfile/index.js (hunk 7 above)
Comment (Justus-Becker-Rabbenstein) [reply pages/singleProfile/index.js]: Fixed.

**Pull request merged**